app/services/IdentityService.scala

```diff
@@ -0,0 +1,30 @@
+package services
+
+import com.typesafe.config.Config
+import models.IdentityId
+import play.api.Logger
+import play.api.libs.json.Json
+import play.api.libs.ws.{WSClient, WSRequest}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class IdentityService(wsClient: WSClient, config: Config)(implicit ec: ExecutionContext) {
+  val identityUrl = config.getString("apiUrl")
+  val token = config.getString("token")
+
+  def request(path: String): WSRequest = {
+    wsClient.url(s"$identityUrl/$path")
+      .withHeaders("Authorization" -> s"Bearer $token")
+  }
+
+  def updateMarketingPreferences(userId: IdentityId, marketing: Boolean): Future[Boolean] = {
+    val payload = Json.obj("statusFields" -> Json.obj("receiveGnmMarketing" -> marketing))
+    request(s"user/${userId.id}").post(payload).map { response =>
+      response.status >= 200 && response.status < 300
+    } recover {
```

Comment:
I think this makes sense for now, but I'd like to see us move to some form of asynchronous communication (Kinesies/SQS/SWF?) for this kind of thing, as we do really want it to happen, but it matters less exactly when it happens.